### PR TITLE
Skip adding comment if there is no change in relevance % of authors

### PR DESCRIPTION
**Pull request by tapishr**
Please check the action items covered in the PR - 

- [x] Build is running
- [ ] Eventing is functional and tested
- [ ] Unit or integration tests added and running
- [x] Manual QA

### CODE REVIEW

File: vibi-dpu/src/core/trigger.rs [hunk 1]
```diff
@@ -50,7 +50,7 @@ pub async fn process_trigger(message_data: &Vec<u8>) {
 	// process_review_changes
 	let hunkmap_opt = process_review_changes(&review).await;
 	// send_hunkmap
-	send_hunkmap(&hunkmap_opt, &review, &repo_config, &access_token).await;
+	send_hunkmap(&hunkmap_opt, &review, &repo_config, &access_token, &None).await;
```
Comment (MuskanPaliwal) [inline vibi-dpu/src/core/trigger.rs:53]: is this a different usecase of `send_hunkmap` function where we are passing old review obj as none?

Comment (MuskanPaliwal) [inline vibi-dpu/src/core/trigger.rs:53]: is this a different usecase of `send_hunkmap` function where we are passing old review obj as none?

**Review by MuskanPaliwal — APPROVED** [review]
A few questions, else it seems good to go

File: vibi-dpu/src/core/relevance.rs [hunk 2]
```diff
@@ -1,196 +1,305 @@
...
+			// add comment
+			if review.provider().to_string() == ProviderEnum::Bitbucket.to_string() {
+				// TODO - add feature flag check
+				// TODO - add comment change check
+				if did_comment_change(&relevance_vec, &old_review_opt) {
+					bitbucket::comment::add_comment(&comment, review, &access_token).await;
+				}
+			}
+			if review.provider().to_string() == ProviderEnum::Github.to_string() {
+				github::comment::add_comment(&comment, review, &access_token).await;
```
[the remark this answers is not in the document]
Comment (tapishr) [reply vibi-dpu/src/core/relevance.rs:35]: Yes, trying it out for bitbucket first

File: vibi-dpu/src/core/trigger.rs (hunk 1 above)
Comment (tapishr) [reply vibi-dpu/src/core/trigger.rs:53]: Yes, this is the trigger functionality. It's not implemented for bitbucket yet and so no sense in sending anything here

**Pull request merged**